i organized all the constants

ipex200 opened the pull request:
i put all the constants required by a certain subsystem in a file for each subsystem so elevator has its own constants ,drive has its own , coralintake has its own ect...

ipex200 committed:
i organized all the constants

i put all the constants required by a certain subsystem in a file for each subsystem so elevator has its own constants ,drive has its own , coralintake has its own ect...

ipex200 committed:
i fixed something

bmalia reviewed (approved):
Looks good to me

MicahCoder commented on src/main/java/frc/robot/subsystems/coralSubsystems/elevator/ElevatorConstants.java:
At this point it feels like it just makes more sense to move Elevator subsystems into the large subsystem folder, distribute the coral constants among CoralIntakeConstants and ElevatorConstants, and lose the coralSubsystems folder.

E-Latsis replied:
I agree; having a CoralConstants file that contains constants for both Elevator and Coral Intake is a bit confusing and moving the subsystems out of the Coral Subsystems folder will make everything a bit more organized and easy to get around.

E-Latsis committed:
CoralIntake and elevator are now their own folder

For organization purposes: Coral Subsystem folder is gone and the elevator and coral Intake files are in the subsystems folder
